nilearn/glm/io.py

```diff
@@ -210,6 +210,12 @@ def save_glm_to_bids(
         check_parameter_in_allowed(key, report_kwargs, "Extra key-word")
         report_kwargs[key] = kwargs[key]
 
+    if (
+        report_kwargs["height_control"] is None
+        and report_kwargs["threshold"] is None
+    ):
+        report_kwargs["threshold"] = 3.09
+
```

Comment:
needed as save_glm_to_bids relies on the deault of generate_report

Reply:
Shouldn't this already be satisfied with the code in `generate_report`?

Reply:
I've checked the above of the code, I see, it is only the defaults of method signature.

Reply:
> Shouldn't this already be satisfied with the code in `generate_report`?

yes but before we get to generate report at the end of `save_glm_to_bids` there is a call `threshold_stat_map` and given that `generate_report` and `threshold_stat_map` have different default threshold value, I felt it was better to set it to actual value early